source/includes/localhost-connection.rst

```diff
@@ -0,0 +1,16 @@
+.. note:: Connect to localhost
```

Comment:
I'm not sure this needs to be a note. I think it'd benefit from being a heading to show in the otp widget.